pymongo/asynchronous/collection.py

```diff
@@ -1176,8 +1188,12 @@ async def replace_one(
             aggregate expression context (e.g. "$$var").
         :param comment: A user-provided comment to attach to this
             command.
+        :param sort: Specify which document the operation updates if the query matches
```

Comment:
Could you add a mention for these new apis in the changelog?

Reply:
Done